tabs-tabs-tabs/manifest.json

```diff
@@ -0,0 +1,17 @@
+{
+  "applications": {
+    "gecko": {
+      "id": "tabs-tabs-tabs@mozilla.org",
+      "strict_min_version": "47.0.0"
```

Comment:
Hmm, that's wierd, all the other addons in this repo have versions like "45.0.0" so I followed that.